GrandCentralBoardTests/Core/ConfigurationRefresherTests.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  Created by Oktawian Chojnacki on 22.04.2016.
+//  Copyright © 2016 Oktawian Chojnacki. All rights reserved.
+//
+
+import XCTest
+import GrandCentralBoardCore
+@testable import GrandCentralBoard
+
+final class ConfigurableMock: Configurable {
+    let expectation: XCTestExpectation
+
+    init(expectation: XCTestExpectation) {
+        self.expectation = expectation
+    }
+
+    func configure(configuration: Configuration) throws {
+        expectation.fulfill()
+    }
+}
+
+public final class ConfigurationDownloaderMock: ConfigurationFetching {
+    public func fetchConfiguration(closure: (Result<Configuration>) -> ()) {
+        closure(.Success(Configuration(builders: [], settings: [])))
```

Comment:
Just thinking...Wouldn't it be better (more realistically) to launch closure asynchronously?